src/js/components/TextArea/doc.js

```diff
@@ -49,17 +49,23 @@ export const themeDoc = {
     type: 'string | (props) => {}',
     defaultValue: undefined,
   },
-  'global.control.border': {
-    description: 'The border of the textarea.',
-    type: 'object',
-    defaultValue: {
-      width: '1px',
-      radius: '4px',
-      color: 'border',
-    },
+  'global.control.border.width': {
+    description: 'The border width.',
+    type: 'string',
+    defaultValue: '1px',
+  },
+  'global.control.border.radius': {
+    description: 'The border radius.',
+    type: 'string',
+    defaultValue: '4px',
+  },
+  'global.control.border.color': {
+    description: 'The border color.',
+    type: 'string',
+    defaultValue: 'border',
   },
   'global.colors.border': {
-    description: 'The color of the borders',
+    description: 'The color of the border',
```

Comment:
Can you please add a dot at the end of the line, it renders better on the README.md file.